sstables/mx/writer.cc

```diff
@@ -1244,10 +1247,12 @@ void writer::write_clustered(const clustering_row& clustered_row, uint64_t prev_
     row_flags flags = row_flags::none;
     row_extended_flags ext_flags = row_extended_flags::none;
     const row_marker& marker = clustered_row.marker();
+    bool is_dead = false;
     if (!marker.is_missing()) {
         flags |= row_flags::has_timestamp;
         if (!marker.is_live() || marker.is_expiring()) {
             flags |= row_flags::has_ttl;
+            is_dead = marker.is_dead(gc_clock::now());
```

Comment:
This is a very small subset of dead rows. Also, a row with a dead marker (or even a completely missing one) can still have live content.

Reply:
So you think a row should be counted as dead in what cases?
- dead row marker and no cells (it might be too expensive to analyse all cells)?
- what about row tombstone?
- what else?

Reply:
A row is dead if it only contains tombstones.

Reply:
It can be as little as a single row tombstone, or as much as thousands of dead cells. Both are valid dead rows. If there is a single live cell, the row is not dead, regardless of how many tombstones it contains.